apps/kyb-app/src/pages/CollectionFlow/versions/v2/components/organisms/CollectionFlowUI/CollectionFlowUI.tsx

```diff
@@ -151,7 +152,7 @@ export const CollectionFlowUI: FunctionComponent<ICollectionFlowUIProps> = ({
         const currentStep = getCollectionFlowState(context)?.steps?.find(
           step => step.stepName === page.stateName,
         );
-        const state = currentStep?.state;
+        const state = currentStep?.state as TCollectionFlowStep['state'];
```

Comment:
Is the type assertion for undefined? It makes sense to have a fallback for undefined instead of an error.